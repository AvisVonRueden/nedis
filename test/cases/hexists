*3
$7
hexists
$11
snowboarder
$4
name
--------
*4
$4
hset
$11
snowboarder
$4
name
$9
guillermo
--------
*3
$7
hexists
$11
snowboarder
$4
blah
--------
*3
$7
hexists
$11
snowboarder
$4
name
--------
*3
$3
set
$3
foo
$3
bar
--------
*3
$7
hexists
$3
foo
$3
bar
