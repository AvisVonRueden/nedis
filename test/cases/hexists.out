:0
:1
:0
:1
+OK
-ERR Operation against a key holding the wrong kind of value
